### DISCUSSION TITLE
Relation Map has been deprecated ？

### DISCUSSION THREAD

**Discussion by jinxiaoman** [TriliumNext/Trilium#4728] [Q&A] [answered]
Relation Map has been deprecated since Trilium 0.63 and will be removed in a future version. Migrate your content to some other note type (e.g. canvas) as soon as possible.

**Comment by zadam** [comment, marked as answer]
Yes, just like the warning says. It hasn't really seen new development for years, is based on an old library which is sometimes buggy, isn't that useful etc.

**Comment by dogfuntom** [comment]
No longer true, see: <https://github.com/zadam/trilium/discussions/4779>.
